.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+
+<!-- 
+
+Sorry to hear you're having trouble! If you have a question, please ask in 
+[SO or gitter](https://github.com/callemall/material-ui/blob/master/CONTRIBUTING.md#asking-questions)
```

Comment:
I know it won't be rendered, but I thought it gave the links context? I like the idea of shortened links. Any preference on shortener?